pkg/kubelet/lifecycle/handlers_test.go

```diff
@@ -25,54 +25,52 @@ import (
 	"testing"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Was this change intentional or automatically generated by the tooling?

Reply:
go language server does whatever it wants, i've disabled it and removed this.